talktorials/11_online_apis/11_intro.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>In order to have a bit more content in this table of content, I suggest to add the subsections here. </p><p>I am not sure how to do this most efficiently for you, I just paste the Markdown code here for you to paste if you like the suggestion:</p><p> </p><p><code>## Learning goals</code></p><p> </p><p><code>### Theory</code></p><p> </p><p><code>Types of programmatic access to web services in Python:</code></p><p><code>- Programmatic APIs</code></p><p><code>- HTTP-based RESTful APIs</code></p><p><code>- Generating a client for any API</code></p><p><code>- Document parsing</code></p><p><code>- Browser remote control</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>